test/e2e/singlecluster/metrics_test.go

```diff
@@ -88,11 +157,39 @@ var _ = ginkgo.Describe("Metrics", func() {
 			}, util.LongTimeout).Should(gomega.Succeed())
 		})
 
+		metrics := []string{
```

Comment:
I would propose extending this list to cover all possible metrics. Otherwise, if an uncovered metric is removed from metrics.Register(), all tests will still pass.

Reply:
Have added more metrics, but would propose to add others in a separate PR covering special cases